diff --git a/node_modules/react-native/Libraries/Components/Touchable/TouchableBounce.js b/node_modules/react-native/Libraries/Components/Touchable/TouchableBounce.js
index 4fcbe80..0a0103a 100644
--- a/node_modules/react-native/Libraries/Components/Touchable/TouchableBounce.js
+++ b/node_modules/react-native/Libraries/Components/Touchable/TouchableBounce.js
@@ -203,6 +203,10 @@ class TouchableBounce extends React.Component<Props, State> {
     this.state.pressability.configure(this._createPressabilityConfig());
   }
 
+  componentDidMount(): mixed {
+    this.state.pressability.configure(this._createPressabilityConfig());
+  }
+
   componentWillUnmount(): void {
     this.state.pressability.reset();
   }
diff --git a/node_modules/react-native/Libraries/Components/Touchable/TouchableHighlight.js b/node_modules/react-native/Libraries/Components/Touchable/TouchableHighlight.js
index 998a7d0..f2f9c4e 100644
--- a/node_modules/react-native/Libraries/Components/Touchable/TouchableHighlight.js
+++ b/node_modules/react-native/Libraries/Components/Touchable/TouchableHighlight.js
@@ -363,6 +363,7 @@ class TouchableHighlight extends React.Component<Props, State> {
 
   componentDidMount(): void {
     this._isMounted = true;
+    this.state.pressability.configure(this._createPressabilityConfig());
   }
 
   componentDidUpdate(prevProps: Props, prevState: State) {
diff --git a/node_modules/react-native/Libraries/Components/Touchable/TouchableNativeFeedback.js b/node_modules/react-native/Libraries/Components/Touchable/TouchableNativeFeedback.js
index 5c5aa5a..83f02dc 100644
--- a/node_modules/react-native/Libraries/Components/Touchable/TouchableNativeFeedback.js
+++ b/node_modules/react-native/Libraries/Components/Touchable/TouchableNativeFeedback.js
@@ -339,6 +339,10 @@ class TouchableNativeFeedback extends React.Component<Props, State> {
     this.state.pressability.configure(this._createPressabilityConfig());
   }
 
+  componentDidMount(): mixed {
+    this.state.pressability.configure(this._createPressabilityConfig());
+  }
+
   componentWillUnmount(): void {
     this.state.pressability.reset();
   }
diff --git a/node_modules/react-native/Libraries/Components/Touchable/TouchableOpacity.js b/node_modules/react-native/Libraries/Components/Touchable/TouchableOpacity.js
index 95b3787..789f039 100644
--- a/node_modules/react-native/Libraries/Components/Touchable/TouchableOpacity.js
+++ b/node_modules/react-native/Libraries/Components/Touchable/TouchableOpacity.js
@@ -314,6 +314,10 @@ class TouchableOpacity extends React.Component<Props, State> {
     }
   }
 
+  componentDidMount(): void {
+    this.state.pressability.configure(this._createPressabilityConfig());
+  }
+
   componentWillUnmount(): void {
     this.state.pressability.reset();
   }
diff --git a/node_modules/react-native/Libraries/Components/Touchable/TouchableWithoutFeedback.js b/node_modules/react-native/Libraries/Components/Touchable/TouchableWithoutFeedback.js
index 8d6a478..63112e7 100755
--- a/node_modules/react-native/Libraries/Components/Touchable/TouchableWithoutFeedback.js
+++ b/node_modules/react-native/Libraries/Components/Touchable/TouchableWithoutFeedback.js
@@ -189,6 +189,10 @@ class TouchableWithoutFeedback extends React.Component<Props, State> {
     this.state.pressability.configure(createPressabilityConfig(this.props));
   }
 
+  componentDidMount(): mixed {
+    this.state.pressability.configure(createPressabilityConfig(this.props));
+  }
+
   componentWillUnmount(): void {
     this.state.pressability.reset();
   }
